test/lib/BedrockTester.cpp

```diff
@@ -576,7 +576,7 @@ bool BedrockTester::readDB(const string& query, SQResult& result, bool online)
         SData command("Query");
         command["Query"] = fixedQuery;
         command["Format"] = "JSON";
-        auto commandResult = executeWaitMultipleData({command});
+        auto commandResult = executeWaitMultipleData({command}, 1);
```

Comment:
This creates 10 connections by default and we only need one. Normally it works fine, but it's a waste of effort. IF we get into a state where the connections are failing, we fail 10 connections instead of 1 which is noisy.